docs/competition/ros_communication.rst

```diff
@@ -607,6 +610,19 @@ Message Definitions
       - :msg:`faulty_part`: True if a part is faulty, False otherwise
       - :msg:`incorrect_part_type`: True if a part has the wrong type, False otherwise
       - :msg:`incorrect_part_color`: True if a part has the wrong color, False otherwise
+    
+    ariac_msgs/msg/AssemblyState
+      .. code-block:: text
+        
+        bool battery_attached
+        bool pump_attached
+        bool sensor_attached
+        bool regulator_attached
+
+      - :msg:`battery_attached`: True if battery is inserted to assembly station, False otherwise
```

Comment:
can you change the wording to "True if battery is attached to the assembly insert, False otherwise"? For all four.

Reply:
Yup done.